### ISSUE TITLE
Ninja-standalone sending duplicated date string in the HTTP response header

### ISSUE THREAD

**Issue by zloster** [ninjaframework/ninja#656]
Hi, all.
There seems to be a problem with the date HTTP header response field. It contains the string formatted date twice.
>{'Content-Length': '31', 'Expires': 'Thu, 01 Jan 1970 00:00:00 GMT', 'Server': 'Jetty(9.4.z-SNAPSHOT)', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Date': **'Sat, 13 Apr 2019 17:06:46 GMT, Sat, 13 Apr 2019 17:06:46 GMT**', 'Content-Type': 'application/json;charset=utf-8'}

Here is an [example](https://travis-ci.org/zloster/FrameworkBenchmarks/jobs/519700502#L1644-L1649).

Affected versions are at least 6.2.2 and 6.4.2. The code producing the above behaviour [is here](https://github.com/zloster/FrameworkBenchmarks/tree/zloster-patch-ninja/frameworks/Java/ninja-standalone).

**Comment by zloster** [comment]
Some more information:
I've spend some time debugging this and I've narrowed the problem to `NinjaServletFilter` and `NinjaServletContext`. I'm attaching 2 (two) screenshots: before the request processing and after this.
![BeforeRequest-2019-05-06 14:05:43](https://user-images.githubusercontent.com/778924/57223309-46220b00-700e-11e9-8cfe-86faa3e4c9b4.png)
![AfterRequest-2019-05-06 14:08:01](https://user-images.githubusercontent.com/778924/57223329-54702700-700e-11e9-8ffd-63d50a71c199.png)

Before the call inside of the filter the HTTP response object already has the `Date` header set (probably Jetty functionality). This value is not propagated further down in Ninja. The headers in the map are empty (ninja.Result class). When the processing is finished there are 2 (two) different keys for the `Date`: one coming from Jetty and one set by Ninja framework.

The second part of the problem is `ServletContextFilter` and this code: https://github.com/ninjaframework/ninja/blob/d8d32d7150270fc65adad800b097e3d65f91b725/ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java#L368-L371
Here the `addHeader()` method is used. This allows for several different values for given HTTP header. This is exactly what I've observed for the `Date` header.

I'm not sure what is the best approach to remedy this:

1. Change `addHeader()` to `setHeader()` and loose the ability to have multiple values per HTTP header;
1. Remove the HTTP date setting from the code;
1. Populate the Ninja Result object with the HTTP headers from the response before processing;
1. Or some combination of the above?
